app/models/pirate.rb

```diff
@@ -1,2 +1,16 @@
 class Pirate
-end
\ No newline at end of file
+  attr_accessor :name, :weight, :height
+
+  @@prirates = []
+
+  def initialize(name, weight, height)
+    @name = name
+    @weight = weight
+    @height = height
+    @@prirates << self
+  end
```

Comment:
The other option that you have here would be to pass one argument here, like `params`, and pull the attributes out of that, just like you did in `Ship`.

```suggestion
  def initialize(params)
    @name = params[:name]
    @weight = params[:weight]
    @height = params[:height]
    @@prirates << self
  end
```